account_commission/models/account_move.py

```diff
@@ -199,6 +199,7 @@ class AccountInvoiceLineAgent(models.Model):
     currency_id = fields.Many2one(
         related="object_id.currency_id",
         readonly=True,
+        store=True,
```

Comment:
This will be fatal on module upgrade, needing to store it for all the existing lines. Why do you need this?